src/System.Numerics.Vectors/src/System/Numerics/Matrix3x2.cs

```diff
@@ -802,9 +802,9 @@ public override string ToString()
         /// <returns>The hash code.</returns>
         public override int GetHashCode()
         {
-            return M11.GetHashCode() + M12.GetHashCode() +
-                   M21.GetHashCode() + M22.GetHashCode() +
-                   M31.GetHashCode() + M32.GetHashCode();
+            return unchecked(M11.GetHashCode() + M12.GetHashCode() +
```

Comment:
nit: can we use a consistent styles? I mean here you inserted the unchecked inline but the next change in the file src/System.Numerics.Vectors/src/System/Numerics/Matrix4x4.cs you have used unchecked {}